main.go

```diff
@@ -286,6 +290,10 @@ func run(args []string) error {
 	if *metricsURL != "" {
 		logger.Printf("metrics enabled; reporting metrics via POST to %s", *metricsURL)
 	}
+	// Always enable prometheus registry. The overhead should be quite minimal as an in-mem map is updated
+	// with the values.
+	pClient := prometheusmetrics.NewPrometheusProvider(metrics.DefaultRegistry, "ghostunnel", "", prometheus.DefaultRegisterer, 1*time.Second)
```

Comment:
What is the “ghostunnel” string used for here? Depending on how this is used the metrics-prefix (defaults to “ghostunnel”) flag might make more sense here.

Reply:
Fixed.